pom.xml

```diff
@@ -1810,6 +1810,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
             <configuration>
+              <doclint>none</doclint>
               <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Is this change mandatory for building javadocs? 
Aren't these two lines somewhat equivalent? Why do we need both?

Reply:
I thought  this depends on the maven version, but found that this depends on the maven-javadoc-plugin version

https://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html

Reply:
Removed the unnecessary line